packages/opentelemetry-context-zone/package.json

```diff
@@ -64,7 +64,7 @@
     "webpack-merge": "5.7.3"
   },
   "dependencies": {
-    "@opentelemetry/context-zone-peer-dep": "0.18.2",
+    "@opentelemetry/context-zone-peer-dep": "^0.19.0",
```

Comment:
To pin or not to pin... Refs: #2073

Reply:
lerna unpins by default. very frustrating